django_libsass.py

```diff
@@ -42,7 +54,12 @@ def compile(**kwargs):
         INCLUDE_PATHS = get_include_paths()
 
     kwargs = kwargs.copy()
-    kwargs['include_paths'] = (kwargs.get('include_paths') or []) + INCLUDE_PATHS
+    kwargs['include_paths'] = kwargs.get('include_paths', []) + INCLUDE_PATHS
```

Comment:
I was worried about everything that might be evaluated to False, including empty collections. But since the default _is_ an empty collection, I guess it's okay.